vars/buildPlugin.groovy

```diff
@@ -27,7 +27,9 @@ def call(Map params = [:]) {
         String label = config.platform
         String jdk = config.jdk
         String jenkinsVersion = config.jenkins
-        String javaLevel = config.javaLevel
+        if (config.javaLevel != null) {
+            echo 'WARNING: javaLevel is deprecated and should not be used'
```

Comment:
Perhaps something like

```
WARNING: Ignoring deprecated "javaLevel" parameter. This parameter should be removed from your "Jenkinsfile".
```

might make it more clear to someone reading the message that the parameter is in fact dead code while also informing them what specific action they can take to (trivially) fix the warning.